sbi/analysis/conditional_density.py

```diff
@@ -270,11 +293,8 @@ def conditonal_potential(
 
     condition = atleast_2d_float32_tensor(condition)
 
-    # Transform the `condition` to unconstrained space.
-    transformed_condition = theta_transform(condition)
-
     conditioned_potential_fn = ConditionedPotential(
-        potential_fn, transformed_condition, dims_to_sample  # type: ignore
+        potential_fn, condition, dims_to_sample  # type: ignore
```

Comment:
These two lines are the main fix in this PR. The rest is just typo fixing

Reply:
well spotted, but where is the condition transformed then?